src/libraries/Microsoft.Extensions.DependencyInjection/tests/DI.External.Tests/Autofac.cs

```diff
@@ -9,12 +9,20 @@ namespace Microsoft.Extensions.DependencyInjection.Specification
 {
     public class AutofacDependencyInjectionSpecificationTests : SkippableDependencyInjectionSpecificationTests
     {
-        public override bool SupportsIServiceProviderIsService => false;
+        public override bool SupportsIServiceProviderIsService => true;
 
         public override string[] SkippedTests => new[]
         {
             "ScopesAreFlatNotHierarchical",
-            "ServiceScopeFactoryIsSingleton"
+            "ServiceScopeFactoryIsSingleton",
+            // different behavior between net7 and net462
```

Comment:
I don't think we should be skipping these tests, especially on net7.

If we think skipping these tests is the right thing - we should only do it on net462.